.ci/validation.sh

```diff
@@ -692,8 +692,8 @@ no-error-sevntu-checks)
   cd .ci-temp/sevntu.checkstyle/sevntu-checks
   mvn -e --no-transfer-progress -Pno-validations verify  -Dcheckstyle.ant.skip=false \
      -Dcheckstyle.version="${CS_POM_VERSION}" \
-     -Dcheckstyle.configLocation=../../../config/checkstyle_checks.xml \
-     -Dcheckstyle.nonMain.configLocation=../../../config/checkstyle_non_main_files_checks.xml
+     -Dcheckstyle.configLocation=../../../config/checkstyle-checks.xml \
+     -Dcheckstyle.nonMain.configLocation=../../../config/checkstyle-non-main-files-checks.xml
```

Comment:
Why is it not finding the file? Local run fails too..

https://checkstyle.semaphoreci.com/jobs/7fc71ca3-4a6b-4cac-afd7-11ca7d5637b3#L786
```
cannot initialize module SuppressionFilter - Unable to find: config/checkstyle-non-main-files-suppressions.xml
```

Reply:
I am not seeing why, but this is a maven run of `sevntu-checks`. It is not a run of Checkstyle. So it doesn't make sense why we are giving sevntu files from Checkstyle.

Reply:
https://github.com/stoyanK7/checkstyle/blob/cbffbbaca86cf0fc8e85c192fd3feecaebd67971/.ci/validation.sh#L685-L699

Reply:
It seems odd, the error message doesn't say ../../, so I am thinking something is missing in the transfer.

Reply:
It does say it in a different place in the error log though
https://checkstyle.semaphoreci.com/jobs/7fc71ca3-4a6b-4cac-afd7-11ca7d5637b3#L735
```
Caused by: org.apache.tools.ant.BuildException: Unable to create Root Module: config {../../../config/checkstyle-non-main-files-checks.xml}.

```

Reply:
I see the issue now. We are passing sevntu our XML configurations so it can run them. However, our config still references our suppression file naming and not sevntu's file naming. Sevntu still has the old name, and our configuration has the new name. Hence it is not finding the file and failing.

@stoyanK7 You must apply similar changes to sevntu just like in other PRs.

Reply:
I suppose we are talking about those lines the `pom.xml`?
```
    <checkstyle.configLocation>https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-${checkstyle.version}/config/checkstyle_checks.xml</checkstyle.configLocation>
    <checkstyle.nonMain.configLocation>https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-${checkstyle.version}/config/checkstyle_non_main_files_checks.xml</checkstyle.nonMain.configLocation>
    <checkstyle.header>https://raw.githubusercontent.com/checkstyle/checkstyle/master/config/java.header</checkstyle.header>
    <checkstyle.regexp.header>https://raw.githubusercontent.com/checkstyle/checkstyle/master/config/java_regexp.header</checkstyle.regexp.header>
```

Reply:
Got it.

Reply:
we have hardcoded file:
https://github.com/checkstyle/checkstyle/blob/a1731ba2da59d713630e06309697df326cdf77d4/config/checkstyle_non_main_files_checks.xml#L12-L15

we need it to be like:
https://github.com/checkstyle/checkstyle/blob/a1731ba2da59d713630e06309697df326cdf77d4/config/checkstyle_checks.xml#L36-L38

So in this case sevntu can stay on names of file it needs to be.

Reply:
@rnveach We have to merge https://github.com/sevntu-checkstyle/sevntu.checkstyle/pull/989 first, before merging this:
https://checkstyle.semaphoreci.com/jobs/2c087901-7776-40ab-9a1a-30350c48a94d#L639
```
[ERROR] [checkstyle] [ERROR] /home/semaphore/checkstyle/.ci-temp/sevntu.checkstyle/sevntu-checks/config/CheckstyleFormatterForEclipse.xml:1: Filename should be in kebab-case. [kebabCaseFileNamingInConfigFolder]
[ERROR] [checkstyle] [ERROR] /home/semaphore/checkstyle/.ci-temp/sevntu.checkstyle/sevntu-checks/config/checkstyle_non_main_files_suppressions.xml:1: Filename should be in kebab-case. [kebabCaseFileNamingInConfigFolder]
```